ui/appui-react/src/appui-react/ui-items-provider/StandardNavigationToolsUiItemsProvider.ts

```diff
@@ -65,11 +66,14 @@ export class StandardNavigationToolsUiItemsProvider implements UiItemsProvider {
 
     } else /* istanbul ignore else */if (toolbarUsage === ToolbarUsage.ViewNavigation && toolbarOrientation === ToolbarOrientation.Vertical) {
 
+      if (!this.defaultNavigationTools || !this.defaultNavigationTools.vertical || this.defaultNavigationTools.vertical.setupWalkCamera)
+        items.push(ToolbarHelper.createToolbarItemFromItemDef(10, CoreTools.setupCameraWalkTool));
+
       if (!this.defaultNavigationTools || !this.defaultNavigationTools.vertical || this.defaultNavigationTools.vertical.walk)
-        items.push(ToolbarHelper.createToolbarItemFromItemDef(10, CoreTools.walkViewCommand));
+        items.push(ToolbarHelper.createToolbarItemFromItemDef(20, CoreTools.walkViewCommand));
 
       if (!this.defaultNavigationTools || !this.defaultNavigationTools.vertical || this.defaultNavigationTools.vertical.toggleCamera)
-        items.push(ToolbarHelper.createToolbarItemFromItemDef(20, CoreTools.toggleCameraViewCommand));
+        items.push(ToolbarHelper.createToolbarItemFromItemDef(30, CoreTools.toggleCameraViewCommand));
```

Comment:
I think we shouldnt change the priority of `walk` and `toggleCamera`, as this will shift things around if someone explicitly added something at priority 15 to show between those 2. Maybe put the `setupCamera` at 5 by default (Same thing in the other provider) ?

Reply:
Agreed. We should  not change the priority of the tools in the standard providers in case consumers rely on them.

Reply:
ok i can change the priority of the setupcamera to 5.